.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 16.x, 18.x, 19.x, 20.x]
+        node-version: [16.x, 18.x, 20.x]
```

Comment:
Old idea: perhaps it would make sense to run CI checks on 16.10 as well? This is the lowest supported version. The 16.x tag resolves to the latest version in the 16 series.